dev/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM debian:sid
```

Comment:
This should probably be based on one of the mediawiki images, and simply add things such as xdebug etc.